packages/react-app/src/constants/Bankless.ts

```diff
@@ -1,10 +1,17 @@
-import { CustomerProps } from '../models/Customer';
+import { CustomerProps } from "../models/Customer";
 /**
```

Comment:
Might want to be careful about linting changes like this - it introduces unecessary code changes